content/docs/1.13/deployment.md

```diff
@@ -4,8 +4,11 @@ weight: 7
 children:
 - title: Operator for Kubernetes
   url: operator
+  navtitle: K8S Operator
```

Comment:
`s/K8S/Kubernetes/`. If there's not much space, then just `s/K8S Operator/Kubernetes/`, as the operator is the preferred way to deploy on Kubernetes anyway.